FRONTEND/newFrontEnd/apTime/src/app/components/notification/notifications/notifications.component.ts

```diff
@@ -66,15 +71,52 @@ export class NotificationsComponent implements OnInit, OnDestroy {
   }
 
   cancel(alert: Notification) {
-    console.log(alert.id);
+    if (alert) {
+      console.log(alert.id);
+      const headers = {
+        Authorization: 'Bearer ' + this.sessionService.getToken(),
```

Comment:
One tiny suggestion: maybe we can define a global variable for the string `'Bearer  '`?

Reply:
it can't be done because it is async and the variable will be out of scope.